### DISCUSSION TITLE
How to enable a dropdown list for switch the same file from different project?

### DISCUSSION THREAD

**Discussion by guyongqiangx** [oracle/opengrok#4186] [Q&A] [answered]
Hi,

I find that at http://aospxref.com/android-11.0.0_r21/xref/system/, there's a dropdown list at the top-left of page-navi bar for all of the projects in this site.
And we can try to switch to view the same file in different projects by selecting the project name in this dropdown list.

<img width="186" alt="3a7e7f4804ff0be911bc61c0307fafc" src="https://user-images.githubusercontent.com/22298198/221462967-752a6085-d1e3-4273-b1d9-2968d987fc8e.png">

For example, now we are viewing file:
http://aospxref.com/android-11.0.0_r21/xref/system/core/fs_mgr/README.overlayfs.md

If we select the item "android-12.0.0_r3", then it will switch to the same file in another project "android-12.0.0_r3", and the web url will be changed to:
http://aospxref.com/android-12.0.0_r3/xref/system/core/fs_mgr/README.overlayfs.md

Now I'm using the latest version v1.7.42, there's only a link of "Home" instead of a dropdown list.
How can I enable the dropdown list in this version?

Thanks

**Comment by guyongqiangx** [comment]
This feature is very useful when I want to view the same file in different version because I create a project for each version of the source and index them separately.

**Comment by vladak** [comment, marked as answer]
This is not something that mainline OpenGrok supports. It is a customization of the site.

**Reply by guyongqiangx** [reply]
OK, thanks for confirmation, do we have any plan to support similar feature?

**Reply by vladak** [reply]
Not on my end of things.
